### ISSUE TITLE
CommandHandler::ProcessCommandDataElement broken for group paths

### ISSUE THREAD

**Issue by bzbarsky-apple** [project-chip/connectedhomeip#8645]
#### Problem
`CommandHandler::ProcessCommandDataElement` does:
```
    err = commandPath.GetEndpointId(&endpointId);
    SuccessOrExit(err);
```
but this will fail for group paths.  What should instead happen is that if there is no endpoint id we check for a group id (which is NOT in the command path; it's in the packet header, iirc), and then loop over all endpoints in the group doing the command on each endpoint.  

 #### Proposed Solution
Fix things so group commands work, like they used to work in the Ember code.

@yunhanw-google @erjiaqing

**Comment by yunhanw-google** [comment]
OK, we would handle it, we also need to remove group id from IM command path in MessageDef

**Comment by bzbarsky-apple** [comment]
@mrjerryjohns should probably confirm what the encoding side looks like before we start changing it.

**Comment by bzbarsky-apple** [comment]
Fixed in #13833.
